content/en/docs/reference/glossary/ingress.md

```diff
@@ -16,5 +16,5 @@ tags:
 
 <!--more--> 
 
-Ingress can provide load balancing, SSL termination and name-based virtual hosting.
+Ingress may provide load balancing, SSL termination and name-based virtual hosting.
```

Comment:
Hello @steadysupply .
What do you think about changing Line 19 to:
Ingress provides load balancing, SSL termination, and name-based virtual hosting.

Reply:
Is an ingress required to provide eg. SSL termination?

Reply:
Hello @steadysupply .
The ingress described in the glossary is the Kubernetes API object, Ingress.
Do you agree with the content in the concept page:
https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource

`An Ingress can be configured to give Services externally-reachable URLs, load balance traffic, terminate SSL / TLS, and offer name based virtual hosting.`

Reply:
Hello @kbhawkey, the same issue applies there (see e609bcd). Part of this issue is asking for clarification (and clarity) on what the Ingress object is _for_.

Reply:
How about:
```markdown
You can use an _Ingress_ to expose your HTTP-based workload to the public
internet. Ingress also lets you configure and use: traffic load balancing,
TLS / SSL termination, and name-based virtual hosting. An Ingress
{{< glossary_tooltip text="controller" term_id="controller" >}} is responsible
for managing resources inside and outside your cluster to set up the Ingress
to match how you configure it.
```
?

Reply:
I'd be happy to see the glossary definition amended.